### ISSUE TITLE
getLUT() computes the wrong number of entries

### ISSUE THREAD

**Issue by ccprog** [Pomax/bezierjs#165]
```js
> bez = new Bezier(1,0,0,2,-1,0)
Bezier { ... }
> bez.getLUT(2)
[ { x: NaN, y: NaN, t: NaN } ]
> bez.getLUT(3)
[ { x: 1, y: 0, t: 0 }, { x: -1, y: 0, t: 1 } ]
> bez.getLUT(4)
[ { x: 1, y: 0, t: 0 }, { x: 0, y: 1, t: 0.5 }, { x: -1, y: 0, t: 1 } ]
```
From the description, I would expect the number of table entries to be `steps+1`, not `steps-1`:

> If `steps` is given, the LUT will contain `steps+1` coordinates representing the coordinates from `t=0` to `t=1` at interval `1/steps`.

Looking at the source, I really wonder what you want to achieve:

```js
getLUT(steps) {
    //...
    // We want a range from 0 to 1 inclusive, so
    // we decrement and then use <= rather than <:
    steps--;
    for (let i = 0, p, t; i < steps; i++) {
      t = i / (steps - 1);
      //...
    }
    //...
  }
```

My understanding would be to do this:

```js
    for (let i = 0, p, t; i <= steps; i++) {
      t = i / steps;
      //...
    }
```

For `steps = 2`, this would produce three entries at `t = 0, 0.5, 1`.

**Comment by Pomax** [comment]
I'll be honest: this is not a number of LUT points that even remotely makes sense, given that your start and end coordinates are already your LUT[2]... what were you trying to do when you ran into this? =D

**Comment by ccprog** [comment]
The above numbers are a test case, not a real use case. It came up for an animation application where the precise number of steps is important.

> The table is supposed to be steps coordinates. Not steps-1, nor steps+1: if you call getLUT(100) you're supposed to get an array with 100 elements. Not 99 or 101, so yeah that definitely needs some fixing.

This actually depends on your definition what the "step" is: the point you are determining, or the way between two points. Typically, if you are describing an animation in **n** steps, you mean **n + 1** points and **n** animation segments. Again, from your [description](https://pomax.github.io/bezierjs/#getLUT) I was expecting this definition to be intended.

**Comment by Pomax** [comment]
Ah, I see - yes if the code conflicts with the docs, that code needs to be updated.

**Comment by Pomax** [comment]
Published v5.0.0 over on https://www.npmjs.com/package/bezier-js (major version update since this is a breaking code change)
